common/gpu/intel/default.nix

```diff
@@ -1,23 +1,42 @@
-{ config, lib, pkgs, ... }:
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
 
 {
-  options.hardware.intelgpu.loadInInitrd = lib.mkEnableOption (lib.mdDoc
-    "loading `i195` kernelModule at stage 1. (Add `i915` to `boot.initrd.kernelModules`)"
-  ) // {
-    default = true;
-  };
+  options.hardware.intelgpu.loadInInitrd =
+    lib.mkEnableOption (
+      lib.mdDoc "loading `i195` kernelModule at stage 1. (Add `i915` to `boot.initrd.kernelModules`)"
+    )
+    // {
+      default = true;
+    };
+
+  options.hardware.intelgpu.loadXeInInitrd =
```

Comment:
Rather than duplicating the option above, I would rather add
`hardware.intelgpu.driver = "i915";` as an option and make new tiger lake + devices using it.